packages/Python/lldbsuite/test/lang/swift/po/sys_types/main.swift

```diff
@@ -13,6 +13,7 @@ import Cocoa
 
 func main() {
   var num = 22
+  //% self.expect("po num", substrs=["\\n", "\""], matching=False) # Make sure po doesn't escape non-printables.
```

Comment:
Thanks for fixing this!
I'm afraid we still have an issue/regression which started happening when the data formatter changed (also for SmallStrings)

Assume:

```
   1    func main() -> Int {
   2      let tinky = "p\n"
   3      var patatino = false
-> 4      return 123
   5    }
```

```
(lldb) po tinky
"p\n"
```

I would expect here `po` printing `p` (and then a newline). Is that correct?